app/controllers/shop/Shop.php

```diff
@@ -1786,8 +1935,10 @@ public function suggestions($pos = 0)
 
         $analyzed = $this->sma->analyze_term($term);
         $sr = $analyzed['term'];
-        //$option_id = $analyzed['option_id'];
+        $convertedData = $this->getArabicToEnglish($sr);
+        $sr = isset($convertedData['to_words'][0]) ? $convertedData['to_words'][0] : "";
```

Comment:
what will happen if the $sr is in english ? how you make sure the customer search in arabic?